### PR TITLE
[connectors] Move assistant selection into  ephemeral message

### PR DESCRIPTION
**Pull request by tdraier**
fixes: https://github.com/dust-tt/tasks/issues/1500 

## Description

Move assistant selection into  ephemeral message.
The tricky part is about sending the bot message metadata to the interaction webhook (which was previously directly passed, as the action was on it, and now it is on a different message) - only the value and the blockId of the block can be passed to the webhook.

## Deploy Plan

deploy connectors

### CODE REVIEW

File: connectors/src/connectors/slack/chat/stream_conversation_handler.ts [hunk 1]
```diff
@@ -245,6 +255,27 @@ export async function streamConversationToSlack(
           },
           { adhereToRateLimit: false }
         );
+        if (
+          slackUserId &&
+          !slackUserInfo.is_bot &&
+          agentConfigurations.length > 0
```
Comment (flvndvd) [inline connectors/src/connectors/slack/chat/stream_conversation_handler.ts:261]: Can we filter out external users as well?

File: connectors/src/connectors/slack/chat/stream_conversation_handler.ts [hunk 2]
```diff
@@ -245,6 +255,27 @@ export async function streamConversationToSlack(
...
+        ) {
+          await slackClient.chat.postEphemeral({
+            channel: slackChannelId,
+            text: "hello",
```
Comment (flvndvd) [inline connectors/src/connectors/slack/chat/stream_conversation_handler.ts:265]: ??

Comment (tdraier) [reply connectors/src/connectors/slack/chat/stream_conversation_handler.ts:265]: Hello :-)  🙈

File: connectors/src/connectors/slack/chat/stream_conversation_handler.ts (hunk 1 above)
Comment (tdraier) [reply connectors/src/connectors/slack/chat/stream_conversation_handler.ts:261]: As external user can't ask the assistant he won't get there - they will be filtered when calling `notifyIfSlackUserIsNotAllowed`

Comment (flvndvd) [reply connectors/src/connectors/slack/chat/stream_conversation_handler.ts:261]: 👍

**Review by flvndvd — APPROVED** [review]

**Pull request merged**